packages/m2-theme/public/index.php

```diff
@@ -15,7 +15,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, shrink-to-fit=no, viewport-fit=cover">
 
     <!-- Muli font import from Abode -->
-    <link rel="stylesheet" href="https://use.typekit.net/gbk7rfi.css">
+    <link rel="preload" href="https://use.typekit.net/gbk7rfi.css" as="style">
```

Comment:
You preload css file there. But is it included in the DOM after?

Reply:
Addressed this by:
- Loading font file in the PHP template same way, as it's implemented in the storefront HTML template (referencing later in the the `head` node).
- Removed duplicate preload of the muli font in the HTML template (left only more recent version of it, which was added in this PR: https://github.com/scandipwa/scandipwa/pull/2897)